pandas/core/resample.py

```diff
@@ -839,7 +839,24 @@ def interpolate(
         **kwargs,
     ):
         """
-        Interpolate values according to different methods.
+        Interpolate values according to different methods. Note that the points used
+        as anchor points in the interpolation are taken from the original timeseries
+        and thus it can lead to information loss and wrong interpolation!
+        
+        ```
+        # example of 2Hz timeseries resampled with rule="400ms"
+        2023-03-01 07:00:00.000    1.0  <- anchor 0
```

Comment:
This would be better shown in the `Examples` section of the docstring

Reply:
Good suggestion - I introduced that.